src/actions/object.js

```diff
@@ -1,6 +1,16 @@
 import axios from 'axios';
+import bodybuilder from 'bodybuilder';
 import * as ActionTypes from '../constants';
 
+// todo: refactor to de-duplicate this logic from the ./objects.js file
```

Comment:
Okay, great. This is still in progress, so I'll keep in mind that the logic is here too, and we can figure out where to move it.